Source/LinkedInWebViewController.swift

```diff
@@ -49,7 +47,14 @@ class LinkedInWebViewController: UIViewController, UIWebViewDelegate {
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        webView = UIWebView(frame: view.bounds)
```

Comment:
`webView = UIWebView(frame: view.bounds)` this can't make the `webView` cover the whole `view`?